docs/UserGuide.md

```diff
@@ -160,6 +160,11 @@ For those who are not as fast, familiarity with the commands over time will allo
   
 * Except `findexpenses`, `findgroups`, `findcontacts`, parameters for other commands are case-sensitive.
 
+* Do note that the [Contact-related commands](#51-contacts-commands) and
+[Expense-related commands](#53-expense-commands) both use accept an index as an argument to identify which
+contact/expense to modify. On the other hand, [Group-related commands](#52-groups-commands) accept a unique group
```

Comment:
We don't allow for modification of expenses though.
Only modification and deletion of contacts and deletion of expenses

Reply:
okay resolved